athlinks-schema.ttl

```diff
@@ -145,10 +145,10 @@ athlinks:bib
 
 athlinks:pace
   a owl:DataTypeProperty ;
-  rdfs:range schema:DateTime .
+  rdfs:range schema:QuantitativeValue .
```

Comment:
I think we need to solve the ambiguity around athlinks:pace. Not sure the solution yet, but this is too ambiguous as it is.